Dockerfile-openresty

```diff
@@ -49,6 +49,7 @@ ARG LUA_BRIDGE_TRACER_VERSION="9213e1b0c23a0d028093895d290c705680fbf4c5"
 ARG JAEGER_VERSION="v0.4.1"
 ARG LIGHTSTEP_VERSION="v0.7.1"
 ARG ZIPKIN_VERSION="v0.4.0"
+ARG DATADOG_VERSION="v0.2.6"
```

Comment:
Update this to 0.3.0, once https://github.com/opentracing-contrib/nginx-opentracing/pull/62 is merged

Reply:
Updated to 0.3.4